pages/resources.css

```diff
@@ -0,0 +1,9 @@
+/* TODO: determine how to put a gutter in between the resources */
+.resourcesWrapper {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
What if you do: `justify-content: space-between;` and on the card styles, you could do: `margin: 0 20px`

Reply:
Modified slightly to `space-evenly` and included vertical gutter, I think this result works reasonably well:

https://share.getcloudapp.com/P8ul1O8d

Reply:
No need to do shareable links! Every time you commit a new preview is deployed for me to see!

You'll wanna check this out on different viewports... Unfortunately, `space-evenly` [doesn't work in IE11](https://caniuse.com/#search=space-evenly), so make sure you provide a back-up rule for `space-between` despite the ugly gap that sometimes exists at the end!

This is being made more complicated by the fact that the ResourceCard's dimensions aren't constrained correctly according to the visible border space. I'll have to fix that and then the margins should work out.

Can you do specifically `margin: 0 20px` so we don't apply any vertical margins?

Reply:
Yea, sure thing! I've just moved "space-between" to be the fallback setting for if space-evenly doesn't exist, and removed the vertical margin.